output/json.go

```diff
@@ -0,0 +1,74 @@
+package output
+
+import (
+	"encoding/json"
+	"github.com/yarpc/crossdock/execute"
+	"github.com/yarpc/crossdock/plan"
+	"io/ioutil"
+)
+
+var StatusNames = map[execute.Status]string{
+	execute.Success: "success",
+	execute.Failed:  "failed",
+	execute.Skipped: "skipped",
+}
+
+type JsonTestReport struct {
```

Comment:
Lint should catch this... perhaps a good opportunity to make sure we're running that with tests...